src/saving/NewSaveMenu.tscn

```diff
@@ -25,12 +25,12 @@ __meta__ = {
 
 [node name="Label" type="Label" parent="VBoxContainer"]
 margin_right = 1280.0
-margin_bottom = 23.0
```

Comment:
Looks like you didn't include any intentional changes in this file, so could you revert to exclude it from this PR?